src/hermez-account.js

```diff
@@ -81,4 +83,15 @@ module.exports = class HermezAccount {
         tx.fromAx = this.ax;
         tx.fromAy = this.ay;
     }
+
+    /**
+     * Sign withdraw-bjj
+     * @param {Object} inputs - Zk inputs  for the withdraw-bjj circuit
+     * @return {Scalar} Signature parameters.
+     */
+    signWithdrawBjj(inputs) {
```

Comment:
Missing unit test for this functionality in https://github.com/hermeznetwork/commonjs/blob/master/test/hermez-account.test.js